dune

```diff
@@ -599,35 +601,85 @@
   (unbox_free_vars_of_closures.mli
    as
    compiler-libs/unbox_free_vars_of_closures.mli)
-  (.ocamloptcomp.objs/native/internal_assembler.cmx as compiler-libs/internal_assembler.cmx)
-  (.ocamloptcomp.objs/byte/internal_assembler.cmi as compiler-libs/internal_assembler.cmi)
-  (.ocamloptcomp.objs/byte/internal_assembler.cmo as compiler-libs/internal_assembler.cmo)
-  (.ocamloptcomp.objs/byte/internal_assembler.cmt as compiler-libs/internal_assembler.cmt)
-  (.ocamloptcomp.objs/byte/internal_assembler.cmti as compiler-libs/internal_assembler.cmti)
+  (.ocamloptcomp.objs/native/internal_assembler.cmx
+   as
```

Comment:
(Also, this is largely just reformatting the
file -- should it be part of the `fmt` target?)

Reply:
Yeah I'll do a separate PR to format these files.